modules/@amperka/usb-keyboard.js

```diff
@@ -185,37 +185,34 @@ var SHIFT_KEYS = {
   '|': 'BACKSLASH'
 };
 
-
 var tap = function(key, callback) {
   var modifiers = 0;
 
   if (Array.isArray(key)) {
-    // We've got an array of modifiers + key itself.
-    // First combine modifiers
     for (var i = 0; i < key.length - 1; ++i) {
       modifiers |= key[i];
     }
-
-    // Then extract last element as a key
     key = key[i];
   }
-
   if (typeof key === 'string') key = KEY[key];
-
-  // Protect from undefined
   key = key || 0;
 
-  E.sendUSBHID([modifiers,0,key,0,0,0,0,0]);
-  setTimeout(function() { 
-    E.sendUSBHID([0,0,0,0,0,0,0,0]); 
+  E.sendUSBHID([modifiers, 0, key, 0, 0, 0, 0, 0]);
+  setTimeout(function() {
+    E.sendUSBHID([0, 0, 0, 0, 0, 0, 0, 0]);
     if (callback) setTimeout(callback, 10);
   }, 10);
 }
 
-var type = function(txt, callback) {
+var type = function(txt, sendSpeed, callback) {
```

Comment:
Backward incompatible change. Those who pass a callback as the second arg will start to fail. Suggest checking the type of arg 2 to support both signatures.